CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set_target_properties(pegmatite-static PROPERTIES
 	POSITION_INDEPENDENT_CODE true
 	OUTPUT_NAME "pegmatite")
 
-list(APPEND CMAKE_CXX_FLAGS "-std=c++11")
+list(APPEND CMAKE_CXX_FLAGS "-std=c++11 -Wall -Wextra")
```

Comment:
Please don't turn on -Wextra by default - warnings under -Wextra are ones that have a high false positive rate.